fastlane/Fastfile

```diff
@@ -829,11 +857,6 @@ platform :ios do
       FileUtils.cp(source, destination)
     end
 
-    # Replace the `default` directory with the contents of `en-US`
-    default_path = File.join(Dir.pwd, 'metadata', 'default')
-    FileUtils.remove_dir(default_path) if File.directory?(default_path)
-    FileUtils.copy_entry(enUS_path, default_path)
```

Comment:
> Sorry for the long comment, but it also helped me write down my reasoning here because I started to get lost in the various contexts of those different .txt files and their source of truth and the pending FIXMEs and all…

There's nothing like writing things down to make sense of things. Or to make sense of how they do not make sense 🙃 Thank you for sharing. Super useful.